### PR TITLE
Add find a rep serializers and fixups to prep for controller

### PR DESCRIPTION
**Pull request by jvcAdHoc**
## Summary

- add `find_a_rep` feature flag that will be used for the controller
- add `max_per_page` module constant for pagination
- rename `find_with_full_name_similar_to` on representative model to match organization's method
- add a `METERS_PER_MILE` module constant
- add serializers for reps and orgs to be used in accreditation controller

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#66847

## Testing done

- Most of the changes are new code that will be unused until the controller is merged
- Tested the renamed class method in the rails console

## What areas of the site does it impact?
No impact yet, it is all unused.

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature

### CODE REVIEW

File: modules/veteran/app/models/veteran/service/organization.rb [hunk 1]
```diff
@@ -5,7 +5,8 @@ module Veteran
   module Service
     class Organization < ApplicationRecord
       FUZZY_SEARCH_THRESHOLD = 0.3 # pg_search's default
```
Comment (holdenhinkle) [inline modules/veteran/app/models/veteran/service/organization.rb:7]: Can we move the following `organization` and `representative` model constants into `Veteran::Service::Constants`? They values for them are the same in both models.

```
      FUZZY_SEARCH_THRESHOLD = 0.3 # pg_search's default
      DEFAULT_MAX_MILES = 50
      DEFAULT_MAX_DISTANCE = DEFAULT_MAX_MILES * Constants::METERS_PER_MILE
```

File: modules/veteran/app/models/veteran/service/organization.rb [hunk 2]
```diff
@@ -36,6 +37,10 @@ def self.find_within_max_distance(long, lat, max_distance = DEFAULT_MAX_DISTANCE
       def self.find_with_name_similar_to(search_phrase)
         where('word_similarity(?, name) >= ?', search_phrase, FUZZY_SEARCH_THRESHOLD)
       end
+
+      def self.max_per_page
```
Comment (holdenhinkle) [inline modules/veteran/app/models/veteran/service/organization.rb:41]: Can we convert this class method in the `organizations` and `representative` models to a constant in `Veteran::Service::Constants`? Both methods return the same value.

**Review by holdenhinkle — COMMENTED** [review]
Looks great. Just a couple of comments.

**Comment by jvcAdHoc** [comment]
good call outs @holdenhinkle. We don't necessarily need configuration at the model level at this time. I'll DRY those up.

File: modules/veteran/app/models/veteran/service/organization.rb (hunk 2 above)
Comment (jvcAdHoc) [reply modules/veteran/app/models/veteran/service/organization.rb:37]: Done!

File: modules/veteran/app/models/veteran/service/organization.rb (hunk 1 above)
Comment (jvcAdHoc) [reply modules/veteran/app/models/veteran/service/organization.rb]: Done!

File: modules/veteran/app/models/veteran/service/organization.rb (hunk 2 above)
Comment (holdenhinkle) [reply modules/veteran/app/models/veteran/service/organization.rb:37]: Nice!

**Review by holdenhinkle — APPROVED** [review]
Looks great, Jon!

File: modules/veteran/app/models/veteran/service/organization.rb (hunk 1 above)
Comment (holdenhinkle) [reply modules/veteran/app/models/veteran/service/organization.rb]: Awesome!

**Review by ryan-mcneil — APPROVED** [review]
👍

**Pull request merged**